src/devices/bus/vme/vme_mvme120.cpp

```diff
@@ -0,0 +1,466 @@
+	// $F60000-F6003F 68451 MMU, mirrored to $F7FFFF
+	// $F80002-F80003 clear cache bank   2	
+	// $F80004-F80005 clear cache bank 1
+	// $F80006-F80007 clear cache bank 1+2
+	// (above mirrored to $F9FFFF)
+}
+
+void vme_mvme120_device::mvme121_mem(address_map &map)
+{
+	mvme12x_base_mem(map);
+	map(0x000000, 0x07ffff).ram().share(m_localram);
+	map(0x080000, 0xefffff).rw(FUNC(vme_mvme120_card_device::vme_a24_r), FUNC(vme_mvme120_card_device::vme_a24_w)); // VMEbus 24-bit addresses
+	// $F60000-F6003F 68451 MMU, mirrored to $F7FFFF
+	// $F80002-F80003 clear cache bank   2	
+	// $F80004-F80005 clear cache bank 1
+	// $F80006-F80007 clear cache bank 1+2
+	// (above mirrored to $F9FFFF)
+}
+
+void vme_mvme120_device::mvme122_mem(address_map &map)
+{
+	mvme12x_base_mem(map);
+	map(0x000000, 0x01ffff).ram().share(m_localram);
+	map(0x020000, 0xefffff).rw(FUNC(vme_mvme120_card_device::vme_a24_r), FUNC(vme_mvme120_card_device::vme_a24_w)); // VMEbus 24-bit addresses
+	// No MMU, no cache
+}
+
+void vme_mvme120_device::mvme123_mem(address_map &map)
+{
+	mvme12x_base_mem(map);
+	map(0x000000, 0x07ffff).ram().share(m_localram);
+	map(0x020000, 0xefffff).rw(FUNC(vme_mvme120_card_device::vme_a24_r), FUNC(vme_mvme120_card_device::vme_a24_w)); // VMEbus 24-bit addresses
+	// $F80002-F80003 clear cache bank   2	
+	// $F80004-F80005 clear cache bank 1
+	// $F80006-F80007 clear cache bank 1+2
+	// (above mirrored to $F9FFFF)
+}
+
+void vme_mvme120_device::device_start()
+{
+	LOG("%s\n", FUNCNAME);
+}
+
+void vme_mvme120_device::device_reset()
+{
+	LOG("%s\n", FUNCNAME);
+	
+	// First 4 machine cycles, ROM is mapped to the reset vector.
+	address_space &program = m_maincpu->space(AS_PROGRAM);
+	program.install_rom(0x000000, 0x000007, m_sysrom);
+	m_memory_read_count = 0;
+
+	m_rom_shadow_tap = program.install_read_tap(0x000000, 0x000007, "rom_shadow_r",[this](offs_t offset, u16 &data, u16 mem_mask)
+	{
+		rom_shadow_tap(offset, data, mem_mask);
+	});
+
+	ctrlreg_w(0, 0xFF);	// /RESET flips the latch bits to $FF
+}
+
+uint16_t vme_mvme120_device::rom_shadow_tap(offs_t address, u16 data, u16 mem_mask)
+{
+	if ((!machine().side_effects_disabled()) && (m_memory_read_count >= 3))
+	{
+		// delete this tap
+		m_rom_shadow_tap->remove();
+
+		// reinstall ram over the rom shadow
+		m_maincpu->space(AS_PROGRAM).install_ram(0x000000, 0x000007, m_localram);
+	}
+	
+	m_memory_read_count++;
+	return data;
+}
+
+//
+WRITE_LINE_MEMBER(vme_mvme120_device::watchdog_reset)
+{
+	if (state) {
+		LOG("%s: MFP watchdog reset\n", FUNCNAME);
+		machine().schedule_soft_reset();
+	}
+}
+
+WRITE_LINE_MEMBER(vme_mvme120_device::mfp_interrupt)
+{
+	LOG("%s: MFP asserting interrupt\n", FUNCNAME);
+	
+	// MFP interrupts are gated by bit 2 of the control register.
+	if(state && !BIT(m_ctrlreg, 2))
+	{
+		LOG("%s: MFP interrupt triggered 68K IRQ6\n", FUNCNAME);
+		m_maincpu->set_input_line(M68K_IRQ_6, ASSERT_LINE);
+	}
+	if(!state)
+	{
+		m_maincpu->set_input_line(M68K_IRQ_6, CLEAR_LINE);
+	}
```

Comment:
Please choose a style and stick with it.  Here you have K&R style in one function (opening brace at end of control statement line) and Allman style (opening brace on line following control statement) in the very next function.  You’ve also mixed `if(…` (no space between flow control keyword and opening parenthesis of control expression) and `if (…` (space between flow control keyword and opening parenthesis of control expression).